content/en/docs/instrumentation/java/automatic/_index.md

```diff
@@ -83,8 +83,3 @@ for the full list.
 You can pass the `-Dotel.javaagent.debug=true` parameter to the agent to see
 debug logs. Note that these are quite verbose.
 
-## Next steps
-
-After you have automatic instrumentation configured for your app or service, you
-may want to add [Manual Instrumentation](../manual) to collect
-custom telemetry data.
```

Comment:
I'd vote to leave these lines here, possibly tweaking the prose to read:

> After you have ..., you might want to [annotate](...) selected methods or add [manual instrumentation](...) ...

WDYT @cartermp?

Reply:
Agreed